 <?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="QT-UK-LT1395" failures="1" tests="1" name="NegativeTest" time="8.261" errors="0" timestamp="2022-03-21T17:07:48 GMT">
  <testcase classname="com.herokuapp.theinternet.NegativeTest" name="login_error_test" time="8.259">
    <failure type="java.lang.AssertionError" message="Actual message is not the same as expected expected [Your username is invalid!
×] but found [Your username is invalid!
Ã—]">
      <![CDATA[java.lang.AssertionError: Actual message is not the same as expected expected [Your username is invalid!
×] but found [Your username is invalid!
Ã—]
at com.herokuapp.theinternet.NegativeTest.login_error_test(NegativeTest.java:65)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 36 stack frames]]>
    </failure>
  </testcase> <!-- login_error_test -->
</testsuite> <!-- NegativeTest -->
